.github/workflows/agent_test.yml

```diff
@@ -24,13 +24,10 @@ jobs:
       fail-fast: false
       matrix:
         dockerimage:
-          - 'openjdk:8-jdk'
```

Comment:
I included the following information in the corresponding commit comment. That ensures that reasons for this decision are not completely lost in a distant future.

Why did I change those images?
* The images of sgrio are at least two years old.
  See https://hub.docker.com/r/sgrio/java
* Images from openjdk are deprecated.
  See https://hub.docker.com/_/openjdk

eclipse-temurin are the official openjdk builds. Since they are based on
ubuntu (as far as the comitter understands it) there is no reason for
ubuntu specific images in the build matrix anymore.